sentences/en/_common.yaml

```diff
@@ -1,12 +1,30 @@
 language: "en"
 responses:
   errors:
+    # General errors
     no_intent: "Sorry, I couldn't understand that"
+    handle_error: "An unexpected error occurred"
+
+    # Errors for when user is not logged in
     no_area: "Sorry, I am not aware of any area called {{ area }}"
-    no_domain: "Sorry, I am not aware of any {{ domain }} in the {{ area }} area"
-    no_device_class: "Sorry, I am not aware of any {{ device_class }} in the {{ area }} area"
-    no_entity: "Sorry, I am not aware of any device or entity called {{ entity }}"
-    handle_error: "An unexpected error occurred while handling the intent"
+    no_domain: "Sorry, I am not aware of any {{ domain }}"
+    no_domain_in_area: "Sorry, I am not aware of any {{ domain }} in the {{ area }} area"
+    no_device_class: "Sorry, I am not aware of any {{ device_class }}"
+    no_device_class_in_area: "Sorry, I am not aware of any {{ device_class }} in the {{ area }} area"
+    no_device: "Sorry, I am not aware of any device called {{ device }}"
```

Comment:
Technically the error should be `no_entity`, as we do not target `device` with sentences.

I know it's confusing because, for most languages, the spoken error will be "I am not aware of this device" because an entity is a very technical term and should not be used while conversing with a voice assistant.

Nevertheless, it should be `no_entity`

I have no strong opinion wether we should change it back or not. I know a core PR is also pending, and If I had to chose, I would prefer to move fast (this time)